### ISSUE TITLE
The backbone network is Resnet50? 

### ISSUE THREAD

**Issue by 814247846** [Zzh-tju/DIoU-SSD-pytorch#9]
Hi, Zzh-tju. It is nice work in your repository of DIoU-SSD-pytorch, and your results show that the output AP50 score is 0.7884902583981603. We can see some improve to SSD(AP50=0.772 ).
But I find the backbone network  in your model is Resnet50 (SSD is using VGG16). So I  wonder the improvement reason is the backbone network or the DIoU method?
Have you done the experienment using Resnet50 without DIoU method? How about the AP50 score?

**Comment by Zzh-tju** [comment]
The backbone network is ResNet50-FPN. All the IoU-based losses are trained by setting the same hyper-parameters. If you want to get a higher AP, you can choose a better backbone network.

**Comment by 814247846** [comment]
您好，同学。我是电子科技大学的学生。
我主要是想请问一下您，您的DIoU在SSD上有了一定的精度提升，但是您的骨干网络相比SSD发生了修改(vgg16---&gt;resnet50)。
所以我在思索，这个AP50的提升会不会是由于您修改了骨干网络导致。我比较好奇的是，您是否比较了①Resnet50+DIoU与②Resnet不使用DIou 在SSD上的精度对比呢？
多多打扰您啦，不好意思。
祝您生活愉快。

&nbsp;

**Comment by Zzh-tju** [comment]
你是说SL1？没有，我们选取baseline为IoU loss，关于Ln范数损失缺点的讨论已经在Unitbox，GIoU，PolarMask等论文中出现过，因此我们就忽略了SL1。关于主干网络的选取，不必太过纠结选哪个。只要控制所有损失都用同一个主干就行

**Comment by 814247846** [comment]
您好，同学！
请问您的代码为什么在github上找不到了呢？
不好意打扰您啦

**Comment by Zzh-tju** [comment]
@814247846 已恢复
